lms/djangoapps/teams/views.py

```diff
@@ -153,14 +153,15 @@ def get(self, request, course_id):
         )
         topics_data["sort_order"] = sort_order
 
-        # Paginate and serialize team membership data.
-        team_memberships = CourseTeamMembership.get_memberships(user.username, [course.id])
-        memberships_data = self._serialize_and_paginate(
+        user = request.user
+
+        user_teams = CourseTeam.objects.filter(membership__user=user)
+        user_teams_data = self._serialize_and_paginate(
             MembershipPagination,
```

Comment:
Yes, since `MembershipPagination` controls the page size. That name is confusing, though; I'll change it.